### PR TITLE
fix(DB/SmartScript) Force Commander Gorax patrol and Shattered Hand Captains missing idle animation

### PR DESCRIPTION
**Pull request by M4th3m4tic4l**
## Changes Proposed:
-  Adds movement to Gorax and animation to captains

## Issues Addressed:

- https://github.com/azerothcore/azerothcore-wotlk/issues/14582
- https://github.com/azerothcore/azerothcore-wotlk/issues/14581
- https://github.com/chromiecraft/chromiecraft/issues/4768
- https://github.com/chromiecraft/chromiecraft/issues/4769

## Tests Performed:

- 
WoWScrnShot_012623_191645

- 

## How to Test the Changes:

1. Insert the changes in your database and go to the indicated area

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by acidmanifesto — COMMENTED** [review]
Do we have a reference for this pr matching what is done currently on retail? Retail Video? Sniff? TC\Mangos source?

**Comment by M4th3m4tic4l** [comment]
As such I do not rely on any sniffs, only videos:
https://youtu.be/ur7vLQmMddQ?t=320
https://www.youtube.com/watch?v=uzQfVw3gbAw&t=9s
Even in retail it's a bit buggy xd

File: data/sql/updates/pending_db_world/rev_1674777294431128400.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-58200, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58201, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58202, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36'),
+(-58203, 0, 0, 0, 1, 0, 100, 0, 0, 0, 3000, 4000, 0, 5, 36, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Shattered Hand Captain - Out of Combat - Play Emote 36');
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1674777294431128400.sql:8]: Shattered Hand Captains are assigned a SmartAI. Wouldn't this override it and prevent them from casting spells?

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1674777294431128400.sql:8]: There could be a workaround for that by using waypoint_scripts. Set a stationary waypoint with a delay, make them emote every time this waypoint is "reached"
That's the best workaround I found so far :p

Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1674777294431128400.sql:8]: > Shattered Hand Captains are assigned a SmartAI. Wouldn't this override it and prevent them from casting spells?

Guids sai will take priority over entey sai

**Comment by Maelthyrr** [comment]
Hi, everything work fine, except Commander Gorax. To me he is stationary. I've test 2 times.

**Comment by M4th3m4tic4l** [comment]
I think it had to do with MovementType (correct me if I'm wrong).

**Comment by Maelthyrr** [comment]
Always static.

**Comment by Maelthyrr** [comment]
Always in WIP ?

**Comment by Maelthyrr** [comment]
Without news, I close the PR, but feel free to ping me to reopen it.

**Pull request closed without merging**